lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -30,18 +30,18 @@ export default function FilePickerApp({
   const {
     authToken,
     authUrl,
-    courseId,
-    enableLmsFilePicker = false,
-    formAction,
-    formFields,
-    googleClientId,
-    googleDeveloperKey,
-    customCanvasApiDomain,
-    lmsUrl,
-    ltiLaunchUrl,
+    filePicker: {
+      formAction,
+      formFields,
+      canvas: { enabled: enableCanvasFilePicker, ltiLaunchUrl, courseId },
```

Comment:
In the context of the `filePickerApp.js` could we get away with `canvasEnabled` here which mapps cleanly onto the config param names?

Reply:
Done https://github.com/hypothesis/lms/pull/1584/commits/2e73a601e2580642cd45e7b1e3f9e19c23a91006